src/bin/ayb.rs

```diff
@@ -94,8 +108,8 @@ async fn main() -> std::io::Result<()> {
                              .value_parser(ValueParser::new(entity_database_parser))
                              .required(true)
                         )
-                        .arg(arg!(<query> "The query to execute")
-                             .required(true))
+                        .arg(arg!(<query> "The query to execute. If not provided, an interactive session to write queries will be launched")
```

Comment:
```suggestion
                        .arg(arg!(<query> "The query to execute. If not provided, an interactive session to write queries will be launched.")
```

Reply:
I have seen that other arguments/flags/subcommands descriptions don't end with a dot. Should we proceed with that change anyways?

Reply:
Thank you for pushing for consistency! :) While `ayb` doesn't have copy guidelines, the rule I've used in other projects/presentations is that if you have multiple sentences, to end the last one in a period. Since we've got `The query to execute.`, the second sentence would end in a period. Not a strict requirement, but figured it would be good here. I'll make the change after merging :).